core/modules/filter/js/filter.js

```diff
@@ -192,6 +192,12 @@ Backdrop.behaviors.editorImageDialog = {
 
       var libraryShown = $('.editor-image-fields').find('[name="attributes[src]"]').is(':visible');
       if (libraryShown) {
+        if ($('ul').hasClass('pager')) {
+          $('li.pager-next a').click(function(){return true;}).click();
```

Comment:
I'm not sure what this click handler is doing. Seems like it's adding a callback that does nothing except for returning "true" and then it's immediately calling it?

If this is needed, this would be better written using `.on('click')` so that it could be removed if needed and prevent double-binding. But maybe we could come up with something else entirely, this seems strange.